src/app/teams/teams.utils.ts

```diff
@@ -1 +1,10 @@
 export const memberCompare = (member1, member2) => member1.userId === member2.userId && member1.userPlanetCode === member2.userPlanetCode;
+
+export function compareFunc(member1, member2) {
+  const name1 = member1.userDoc.doc.lastName;
+  const name2 = member2.userDoc.doc.lastName;
+  if (name1 && name2) {
+    return name1 < name2 ? -1 : 1;
```

Comment:
When sorting strings, you `localeCompare` is very useful because it automatically returns a number that works in a sort function (negative, positive, or 0).